### ISSUE TITLE
(v2) duplicated line in `split-editors` example on init

### ISSUE THREAD

**Issue by meowgorithm** [charmbracelet/bubbletea#1157]
This is likely on the Bubbles v2 side of things but when the `split-editors` example initializes the first lines in `textarea` are duplicated. This goes away on keypress.

<img width="708" alt="Screenshot 2024-09-18 at 3 09 36 PM" src="https://github.com/user-attachments/assets/c6dcd27a-6d81-4c55-b6ab-40095d7408f6">

**Comment by aymanbagabas** [comment]
I'm not able to reproduce using Kitty and Ghostty with/without Tmux :/

**Comment by haoqixu** [comment]
I think #1159 is also caused by the same problem.

**Comment by haoqixu** [comment]
> I think #1159 is also caused by the same problem.

cc @robinovitch61

**Comment by robinovitch61** [comment]
Yes, this seems resolved on the latest v2 alpha. Thanks @haoqixu
